<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Small Bills and Petty Finance DB Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2" rowspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="deleteEntryButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete entry</string>
        </property>
        <property name="shortcut">
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="newEntryButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>New entry</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+N</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="3">
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>577</width>
           <height>659</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="2">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>E&amp;ntered by:</string>
              </property>
              <property name="buddy">
               <cstring>userEntryEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="entryNumEdit">
              <property name="placeholderText">
               <string>Entry number</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="repoComboBox"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Entered on:</string>
              </property>
              <property name="buddy">
               <cstring>dateEntryEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="5">
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="bundleRefEntryEdit"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Repository:</string>
              </property>
              <property name="buddy">
               <cstring>repoComboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QCheckBox" name="headerBox">
              <property name="text">
               <string>Printed header?</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2" colspan="2">
             <widget class="QLineEdit" name="consNotesEdit">
              <property name="placeholderText">
               <string>Conservation notes</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QCheckBox" name="manuBox">
              <property name="text">
               <string>All manuscript?</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLineEdit" name="yearEdit">
              <property name="placeholderText">
               <string>Year</string>
              </property>
             </widget>
            </item>
            <item row="18" column="2">
             <widget class="QLineEdit" name="benifForenameEdit">
              <property name="placeholderText">
               <string>Forename if yes</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Docu&amp;ment type:</string>
              </property>
              <property name="buddy">
               <cstring>docTypeComboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QComboBox" name="docDescBox"/>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Doc description:</string>
              </property>
              <property name="buddy">
               <cstring>docDescBox</cstring>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLineEdit" name="poundsEdit">
              <property name="placeholderText">
               <string>Pounds (Â£)</string>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QCheckBox" name="imageBox">
              <property name="text">
               <string>Contains image?</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QCheckBox" name="consReqBox">
              <property name="text">
               <string>Conservation assessment required?</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="4">
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="18" column="3">
             <widget class="QLineEdit" name="benifSurnameEdit">
              <property name="placeholderText">
               <string>Surname if yes</string>
              </property>
             </widget>
            </item>
            <item row="14" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Paid to:</string>
              </property>
             </widget>
            </item>
            <item row="16" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Paid by:</string>
              </property>
             </widget>
            </item>
            <item row="15" column="0">
             <widget class="QLineEdit" name="toForenameEdit">
              <property name="placeholderText">
               <string>Forename</string>
              </property>
             </widget>
            </item>
            <item row="10" column="2">
             <widget class="QLineEdit" name="penceEdit">
              <property name="placeholderText">
               <string>Pence (p)</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QComboBox" name="monthComboBox"/>
            </item>
            <item row="10" column="3">
             <widget class="QLineEdit" name="penceTotalEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="placeholderText">
               <string>Total in pence</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0" colspan="4">
             <widget class="Line" name="line_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="15" column="1">
             <widget class="QLineEdit" name="toSurnameEdit">
              <property name="placeholderText">
               <string>Surname</string>
              </property>
             </widget>
            </item>
            <item row="12" column="0" colspan="4">
             <widget class="QPlainTextEdit" name="descriptionTextEdit">
              <property name="placeholderText">
               <string>Commodity or service description</string>
              </property>
             </widget>
            </item>
            <item row="13" column="0" colspan="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Commodity or service general category:</string>
              </property>
             </widget>
            </item>
            <item row="17" column="1">
             <widget class="QLineEdit" name="fromSurnameEdit">
              <property name="placeholderText">
               <string>Surname</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QComboBox" name="docTypeComboBox"/>
            </item>
            <item row="10" column="1">
             <widget class="QLineEdit" name="shillingsEdit">
              <property name="placeholderText">
               <string>Shillings (s)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Bundle ref:</string>
              </property>
              <property name="buddy">
               <cstring>bundleRefEntryEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="itemNumEdit">
              <property name="placeholderText">
               <string>Item number</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="userEntryEdit"/>
            </item>
            <item row="17" column="0">
             <widget class="QLineEdit" name="fromForenameEdit">
              <property name="placeholderText">
               <string>Forename</string>
              </property>
             </widget>
            </item>
            <item row="19" column="0" colspan="4">
             <widget class="QLineEdit" name="benifNotesEdit">
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="placeholderText">
               <string>Notes if no</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDateEdit" name="dateEntryEdit">
              <property name="displayFormat">
               <string>yyyy-MM-dd</string>
              </property>
             </widget>
            </item>
            <item row="18" column="0" colspan="2">
             <widget class="QCheckBox" name="benifPersonBox">
              <property name="text">
               <string>Is benificiary a person?</string>
              </property>
             </widget>
            </item>
            <item row="9" column="2">
             <widget class="QLineEdit" name="dayEdit">
              <property name="placeholderText">
               <string>Date</string>
              </property>
             </widget>
            </item>
            <item row="13" column="2">
             <widget class="QLineEdit" name="commodCatAEdit">
              <property name="placeholderText">
               <string>General</string>
              </property>
             </widget>
            </item>
            <item row="13" column="3">
             <widget class="QLineEdit" name="commodCatBEdit">
              <property name="placeholderText">
               <string>Specific</string>
              </property>
             </widget>
            </item>
            <item row="15" column="2">
             <widget class="QLineEdit" name="toJobCatAEdit">
              <property name="placeholderText">
               <string>Sector</string>
              </property>
             </widget>
            </item>
            <item row="15" column="3">
             <widget class="QLineEdit" name="toJobCatBEdit">
              <property name="placeholderText">
               <string>Occupation</string>
              </property>
             </widget>
            </item>
            <item row="17" column="2">
             <widget class="QLineEdit" name="fromJobCatAEdit">
              <property name="placeholderText">
               <string>Sector</string>
              </property>
             </widget>
            </item>
            <item row="17" column="3">
             <widget class="QLineEdit" name="fromJobCatBEdit">
              <property name="placeholderText">
               <string>Occupation</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTableView" name="entriesTable"/>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="newDatabaseButton">
        <property name="text">
         <string>New database...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="saveEntryButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Save database</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="openDatabaseButton">
        <property name="text">
         <string>Open database...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Copyright Â© 2018 Larchwood Research</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>995</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>repoComboBox</tabstop>
  <tabstop>userEntryEdit</tabstop>
  <tabstop>dateEntryEdit</tabstop>
  <tabstop>bundleRefEntryEdit</tabstop>
  <tabstop>itemNumEdit</tabstop>
  <tabstop>entryNumEdit</tabstop>
  <tabstop>docTypeComboBox</tabstop>
  <tabstop>docDescBox</tabstop>
  <tabstop>consReqBox</tabstop>
  <tabstop>consNotesEdit</tabstop>
  <tabstop>manuBox</tabstop>
  <tabstop>headerBox</tabstop>
  <tabstop>imageBox</tabstop>
  <tabstop>yearEdit</tabstop>
  <tabstop>monthComboBox</tabstop>
  <tabstop>dayEdit</tabstop>
  <tabstop>poundsEdit</tabstop>
  <tabstop>shillingsEdit</tabstop>
  <tabstop>penceEdit</tabstop>
  <tabstop>penceTotalEdit</tabstop>
  <tabstop>descriptionTextEdit</tabstop>
  <tabstop>commodCatAEdit</tabstop>
  <tabstop>commodCatBEdit</tabstop>
  <tabstop>toForenameEdit</tabstop>
  <tabstop>toSurnameEdit</tabstop>
  <tabstop>toJobCatAEdit</tabstop>
  <tabstop>toJobCatBEdit</tabstop>
  <tabstop>fromForenameEdit</tabstop>
  <tabstop>fromSurnameEdit</tabstop>
  <tabstop>fromJobCatAEdit</tabstop>
  <tabstop>fromJobCatBEdit</tabstop>
  <tabstop>benifPersonBox</tabstop>
  <tabstop>benifForenameEdit</tabstop>
  <tabstop>benifSurnameEdit</tabstop>
  <tabstop>benifNotesEdit</tabstop>
  <tabstop>newDatabaseButton</tabstop>
  <tabstop>openDatabaseButton</tabstop>
  <tabstop>newEntryButton</tabstop>
  <tabstop>saveEntryButton</tabstop>
  <tabstop>deleteEntryButton</tabstop>
  <tabstop>entriesTable</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
